content/guides/api-calls/permissions-and-errors/scopes.md

```diff
@@ -201,9 +201,16 @@ Governance.
 There are some additional scopes that are available on request which will
 enable some specific capabilities for your application.
 
-This scope can be requested by opening a ticket via our support team, who will
-review request on an individual basis and only provide approval of the use case
-requires the scope.
+These scopes can be requested by opening a ticket via our
+[support team](page://support), who will review requests on an individual
+basis and only provide approval if the use case requires the scope.
+
+<Message type='notice'>
+  Scopes which are only available on request are not available for activation
```

Comment:
Sentence reads a bit strange. Maybe change to: `It is not possible to request extra scopes if your account is a free trial account`?

Reply:
wfm, changes made